### ISSUE TITLE
data directory not rw for tomcat

### ISSUE THREAD

**Issue by froggypaule** [protegeproject/webprotege#660]
Dear all,
Sorry, but I simply can't make sense of it, hence my posting. 
I keep getting "WebProtege is not able to write to the WebProtege data directory located at /srv/webprotege" while I truly have 

xxxx@localhost:~$ sudo ls -l /srv
total 4
drwxrwxrwx 2 root tomcat 4096 Jan 29 13:55 webprotege

(the permissions for o are not necessary of course)

So I must be missing the obvious, but what ? many thanks!
(this concerns webprotege-4.0.0-beta-1)

Paulette.

**Comment by santiagorr** [comment]
Same here. But with webprotege-4.0.0-beta-2:

    ls -ld /srv/webprotege
    drwxr-xr-x 2 tomcat tomcat 4096 Jan 30 09:35 /srv/webprotege

**Comment by santiagorr** [comment]
@froggypaule I found the reason of the issue in my environment (Debian 10, running with systemd). As stated by `/usr/share/doc/tomcat9/README.Debian`,  "Tomcat is sandboxed by systemd" and has only write access to some directory. I needed to create an override with:
`systemctl edit tomcat9.service`
then add:
````   
[Service]
ReadWritePaths=/srv/webprotege/
ReadWritePaths=/var/log/webprotege/
````
and
`systemctl daemon-reload && systemctl restart tomcat9`

Also, I had to use the java from oracle.

HTH
